accounts/account_manager.go

```diff
@@ -97,3 +119,11 @@ func (am *AccountManager) Accounts() ([]Account, error) {
 	}
 	return accounts, err
 }
+
+func unlockLater(am *AccountManager, addr []byte) {
+	time.Sleep(time.Millisecond * time.Duration(am.unlockedMilliSeconds))
```

Comment:
Why is that better?                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Suggestion

select {
    case <-time.After(time.Millisecond * am.unlockMilliseconds):
}‎